packages/react/src/TreeView/TreeView.tsx

```diff
@@ -120,6 +121,12 @@ const UlBox = styled.ul<SxProp>`
         outline: 2px solid transparent;
         outline-offset: -2px;
       }
+
+      .PRIVATE_TreeView-item-drag-handle {
```

Comment:
Is this a defined pattern anywhere else?

Reply:
Not that I'm aware of (regarding this being a defined pattern). 

Since we need to show the drag handle when the user hovers over the `TreeItem`, I decided to toggle the visibility of that class (that contains the drag handle) from the `hover` property of the entire container 🤔 

@dusave

Reply:
Sorry, my comment wasn't clear. I meant was the _naming convention_ a defined pattern anywhere else? I'm thinking we probably don't want to draw attention to something marked as `PRIVATE`